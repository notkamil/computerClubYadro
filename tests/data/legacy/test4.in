1
09:00 10:00
20
09:10 1 client1
09:15 2 client1 1
09:45 4 client1