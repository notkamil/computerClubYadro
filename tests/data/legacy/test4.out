09:00
09:10 1 client1
09:15 2 client1 1
09:45 4 client1
10:00
1 20 00:30